research-hub-web/src/app/components/shared/error-dialog/error-dialog.component.spec.ts

```diff
@@ -0,0 +1,47 @@
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { SharedModule } from '../../../components/shared/app.shared.module';
+import { RouterModule } from '@angular/router';
+import { By } from '@angular/platform-browser';
+import { CommonModule } from '@angular/common';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ErrorDialogComponent } from './error-dialog.component';
+
+// import { ResearchHubApiService } from "../../services/research-hub-api.service";
```

Comment:
Remove commented out code

Reply:
Done. Deleted the error dialog spec file since it had nothing in it.